src/app/containers/Header/a11y.test.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react';
+import { service as newsServiceConfig } from '#lib/config/services/news';
+const bannerMock = require('../ConsentBanner');
+const brandMock = require('../Brand');
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { FRONT_PAGE } from '#app/routes/utils/pageTypes';
+import { RequestContextProvider } from '#contexts/RequestContext';
+import { ServiceContext } from '#contexts/ServiceContext';
+import { ToggleContext } from '#contexts/ToggleContext';
+import HeaderContainer from './index';
+
+const defaultToggleState = {
+  navOnArticles: {
+    enabled: true,
+  },
+  scriptLink: {
+    enabled: true,
+  },
+  variantCookie: {
+    enabled: true,
+  },
+};
+
+const mockToggleDispatch = jest.fn();
+
+jest.mock('../ConsentBanner', () => jest.fn());
+
+jest.mock('../Brand', () => jest.fn());
+
+brandMock.mockImplementation(({ focusRef }) => {
+  return (
+    <a tabIndex="-1" ref={focusRef}>
```

Comment:
tabindex -1 isn't needed on links, links are focusable by default

Reply:
I added tabIndex -1 to allow them to be focusable for the test but I now realise they were just not valid links as they didn't had a `href` attribute, this has now been correct, thanks!